packages/muelu/doc/TutorialReadTheDocs/pages/level_smoothers.rst

```diff
@@ -0,0 +1,174 @@
+
+.. _level_smoothers/figure_1vcycles:
+
+.. list-table:: 2D Laplace equation on 50 x 50 mesh after 1 V-cycle with an AMG multigrid solver and Jacobi smoothers on all multigrid levels (2 processors)
+
+  * - .. figure:: pics/1level_1jac09.png
+
+        1 level with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/1level_10jac09.png
+
+        1 level with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/1level_100jac09.png
+
+        1 level with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+  * - .. figure:: pics/2level_1jac09.png
+
+        2 levels with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/2level_10jac09.png
+
+        2 levels with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/2level_100jac09.png
+
+        2 levels with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+  * - .. figure:: pics/3level_1jac09.png
+
+        3 levels with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/3level_10jac09.png
+
+        3 levels with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/3level_100jac09.png
+
+        3 levels with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+.. _level_smoothers/figure_5vcycles:
+
+.. list-table:: 2D Laplace equation on 50 x 50 mesh after 5 V-cycle with an AMG multigrid solver and Jacobi smoothers on all multigrid levels. (2 processors)
+
+  * - .. figure:: pics/5sweeps_1level_1jac09.png
+
+        1 level with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_1level_10jac09.png
+
+        1 level with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_1level_100jac09.png
+
+        1 level with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+  * - .. figure:: pics/5sweeps_2level_1jac09.png
+
+        2 levels with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_2level_10jac09.png
+
+        2 levels with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_2level_100jac09.png
+
+        2 levels with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+  * - .. figure:: pics/5sweeps_3level_1jac09.png
+
+        3 levels with 1 Jacobi sweep (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_3level_10jac09.png
+
+        3 levels with 10 Jacobi sweeps (:math:`\omega=0.9`)
+
+    - .. figure:: pics/5sweeps_3level_100jac09.png
+
+        3 levels with 100 Jacobi sweeps (:math:`\omega=0.9`)
+
+One has even more fine-grained control over pre- and post-smoothing as shown in the following example,
+where we use different damping parameters for pre- and post-smoothing (and a direct solver on the coarse grid):
+
+.. literalinclude:: ../../../test/tutorial/s1_easy_jacobi2.xml
+  :language: xml
+  :caption:
+
+This produces the following multigrid hierarchy
+
+.. warning::
+
+    Insert missing output
+
+.. note::
+
+    Note that the relaxation-based methods provided by the Ifpack/Ifpack2 package are embedded in an outer additive Schwarz method.
```

Comment:
We're not generally using it in additive Schwarz mode. So I am not sure if this comment doesn't make things more confusing.